docs/sources/tempo/getting-started/tempo-in-grafana.md

```diff
@@ -62,3 +62,14 @@ The JSON data can be downloaded via the Tempo API or the [Inspector panel](/docs
 {{% admonition type="note" %}}
 To perform this action on Grafana 10.1 or later, select a Tempo data source, select **Explore** from the main menu, and then select **Import trace**.
 {{% /admonition %}}
+
+## Link tracing data with profiles
+
+Using Trace to profiles, you can use Grafana’s ability to correlate different signals by adding the functionality to link between traces and profiles.
+
+Trace to profiles lets you link your [Grafana Pyroscope](/docs/pyroscope/) data source to tracing data in Grafana or Grafana Cloud.
+When configured, this connection lets you run queries from a trace span into the profile data.
+
+For more information, refer to the [Traces to profiles documentation](https://grafana.com/docs/grafana/<GRAFANA_VERSION>/datasources/tempo/configure-tempo-data-source#trace-to-profiles) and the [Grafana Pyroscope data source documentation](https://docs/grafana/<GRAFANA_VERSION>/datasources/grafana-pyroscope/).
```

Comment:
The doc-validator issue will be resolved once the other doc-validator Tempo PR is merged.

Reply:
I've just merged https://github.com/grafana/tempo/pull/3331 🙂

Reply:
@jdbaldry I'm going to default to what we have in the Writers' Toolkit which says to use a full URL when linking to grafana.com pages. 
![image](https://github.com/grafana/tempo/assets/104772500/d2bf29eb-521f-4d51-9bf5-ef8180a5b496)

The doc-validator is reporting an issue that I need to use partial URLs.

Reply:
The workflow running on this PR is out of date and is still running `doc-validator` v3.0.0 (https://github.com/grafana/tempo/actions/runs/7658560103/workflow?pr=3330).

Reply:
Probably easiest to ignore it for now, merge this PR and know that future PRs should run with `doc-validator` v4.0.0.